articles/test-runner-plugins.md

```diff
@@ -0,0 +1,236 @@
+<!--
+parent: 'Developer Guide'
+toc: true
+skip : true
+authors:
+    - 'Bertrand Chevrier'
+tags:
+    - 'Frontend'
+    - 'Plugin'
+    - 'JavaScript'
+    - 'Test Runner'
+-->
+Enhance the user experience with Test Plugins
+=============================================
+
+One of the best option to improve, customize the test taker experience is using _Test Runner Plugins_.
```

Comment:
Summary is missing. You can easily fix this by adding a > in front of this paragraph